src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  height: 408px;
+  border: 1px;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__text {
+    color: $primarycolor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    color: $secondarycolor;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    margin: 4px 84px 0 16px;
+  }
+
+  &__rating {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin: 16px 16px 0;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    margin-left: 17px;
```

Comment:
Use correct `justify-content` on parent of this element, instead moving it with a fixed margin